R/infrastructure-git.R

```diff
@@ -190,10 +241,12 @@ use_git_ignore <- function(ignores, directory = ".", pkg = ".") {
 #' those fields already exist.
 #'
 #' @inheritParams use_git
+#' @param host GitHub host to use. Override with your GitHub enterprise
+#'    hostname, for example, "https://github.hostname.com".
 #' @family git infrastructure
 #' @keywords internal
 #' @export
-use_github_links <- function(pkg = ".") {
+use_github_links <- function(pkg = ".", host = "https://github.com") {
```

Comment:
Ah you are right, my apologies, I think appending '/issues' to `html_url` is the right thing then.